iphone/Classes/TiContactsPerson.h

```diff
@@ -31,10 +37,16 @@
 
 -(id)_initWithPageContext:(id<TiEvaluator>)context recordId:(ABRecordID)id_ module:(ContactsModule*)module_;
 -(id)_initWithPageContext:(id<TiEvaluator>)context person:(ABRecordRef)person_ module:(ContactsModule*)module_;
+
 @property(readonly,nonatomic) NSString* identifier;
+@property(assign, nonatomic) id<TiContactsPersonUpdateObserver> observer;
+
 +(NSDictionary*)iOS9multiValueLabels;
 +(NSDictionary*)iOS9propertyKeys;
--(id)_initWithPageContext:(id<TiEvaluator>)context contactId:(CNMutableContact*)person_ module:(ContactsModule*)module_;
```

Comment:
This is the initializer I wanted to stay there (with no observer), so modules can still use it.